libethashseal/genesis/mainNetworkTest.cpp

```diff
@@ -56,6 +56,9 @@ R"E(
 		"0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
 		"0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } },
 		"0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" } }
+		"0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } } },
```

Comment:
Metropolis fork block is here "0x2dc6c0", so these and modexp should probably have `startingBlock` with this number